include/openssl/mdc2.h

```diff
@@ -18,31 +18,37 @@
 
 # include <openssl/opensslconf.h>
 
-#ifndef OPENSSL_NO_MDC2
-# include <stdlib.h>
-# include <openssl/des.h>
-# ifdef  __cplusplus
+# ifndef OPENSSL_NO_MDC2
+
+#  define MDC2_DIGEST_LENGTH      16
```

Comment:
But here you already decided not to deprecate!

Reply:
I had a reason but don't remember why at the moment and can't figure it out.